src/traces/parcoords/parcoords.js

```diff
@@ -569,18 +572,7 @@ module.exports = function(root, svg, styledData, layout, callbacks) {
                     .outerTickSize(2)
                     .ticks(wantedTickCount, d.tickFormat) // works for continuous scales only...
                     .tickValues(d.ordinal ? // and this works for ordinal scales
-                        sdom.map(function(dd, i) {
-                            if(texts) {
-                                var text = texts[i];
-                                if(text === null || text === undefined) {
-                                    return dd;
-                                } else {
-                                    return text;
-                                }
-                            } else {
-                                return dd;
-                            }
-                        }) :
+                        sdom.map(toText(Lib.identity), texts) :
```

Comment:
Am I missing someting or are the `()` not in the right place here? 
`sdom.map(toText(Lib.identity, texts))`